runtime/onert/backend/cpu/ops/ConvolutionLayer.h

```diff
@@ -53,11 +53,14 @@ class ConvolutionLayer : public ::onert::exec::IFunction
   void convQuant8();
 
   void configure(const Tensor *input, const Tensor *kernel, const Tensor *bias,
-                 const ir::PaddingType paddingType, const uint32_t paddingLeft,
-                 const uint32_t paddingRight, const uint32_t paddingTop,
-                 const uint32_t paddingBottom, const uint32_t strideW, const uint32_t strideH,
+                 const ir::Padding padding, const uint32_t paddingLeft, const uint32_t paddingRight,
+                 const uint32_t paddingTop, const uint32_t paddingBottom,
+                 const uint32_t strideWidth, const uint32_t strideHeight,
                  const ir::Activation activation, Tensor *output);
-
+  void configure(const Tensor *input, const Tensor *kernel, const Tensor *bias,
+                 const ir::Padding padding, const ir::Activation activation,
+                 const uint32_t strideWidth, const uint32_t strideHeight,
+                 const ir::Layout current_op_seq_layout, Tensor *output);
```

Comment:
@hseok-oh or @ragmani Could you please review the change in `ConvolutionLayer.h`?

Reply:
I hope this wouldn't be introduced. `current_op_seq_layout` is frontend layout. But `Tensor` has backend layout.